rpcs3/Emu/Cell/lv2/sys_net.cpp

```diff
@@ -214,6 +213,17 @@ static sys_net_error get_last_error(bool is_blocking, int native_error = 0)
 		ERROR_CASE(ECONNRESET);
 		ERROR_CASE(EADDRINUSE);
 		ERROR_CASE(EISCONN);
+		ERROR_CASE(EINVAL);
+		ERROR_CASE(EINTR);
+		ERROR_CASE(EFAULT);
+#ifdef _WIN32
+		ERROR_CASE(ENETDOWN);
+		ERROR_CASE(EOPNOTSUPP);
+		ERROR_CASE(ESHUTDOWN);
+		ERROR_CASE(EMSGSIZE);
+		ERROR_CASE(ECONNABORTED);
+		ERROR_CASE(ETIMEDOUT);
+#endif
```

Comment:
This entire commit doesnt make any sense.. those are guest error codes for guest errors. they are **not** sys_net errors and shouldnt be treated as such.
sys_net errors are constant and shouldnt change between different platform as well..
Most of the errors here are already handled in actual sys_net code and not by the operating system..

Reply:
Part of this converts OS error codes into SYS_NET error codes.
So if we receive an OS error code that has an equivalent SYS_NET error code, then we should be doing the translation.